test/float/run_tests.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+mydir = os.path.dirname(__file__)
+os.chdir(mydir)
+sys.path.insert(0, os.path.realpath('..'))
+
+from sailtest import *
+
+sail_dir = get_sail_dir()
+sail = get_sail()
+test_cases_dir = "cases"
+
+print("Sail is {}".format(sail))
+print("Sail dir is {}".format(sail_dir))
+
+def test_float(name):
+    banner('Testing floating point')
+    results = Results(name)
+    for filenames in chunks(os.listdir(test_cases_dir), parallel()):
+        tests = {}
+        for filename in filenames:
+            basename = os.path.splitext(os.path.basename(filename))[0]
+            tests[filename] = os.fork()
+            if tests[filename] == 0:
+                step('{} {}/{} 2> {}.error'.format(sail, test_cases_dir, filename, basename))
```

Comment:
Hi @Alasdair 

Could you please help to enlighten me the best practice to run the `nan_test.sail` here when you are free?
Just work out a POC of the standalone float lib but would like to consult from expert like you for the right direction.

Thanks and CC @billmcspadden-riscv for awareness.

Reply:
Hi, I'll take a look next week. Please ping me again if I forget and don't get around to it.

Reply:
Sure thing and thanks a lot. It is not urgent, take your time.

Reply:
Kindly ping for this, take your time if you are in the middle of sth.